index.html

```diff
@@ -0,0 +1,107 @@
+<!DOCTYPE html>
+<html lang="en">
+   <head>
+      <meta charset="UTF-8" />
+      <title>Fork Me! FCC: Test Suite Template</title>
+      <!-- google fonts link  -->
+      <link
+         href="https://fonts.googleapis.com/css2?family=Bungee&family=Dosis:wght@200;400&family=Inter:wght@400;600;700&family=Karla:wght@400;500;600;700&family=Montserrat:wght@100&family=Open+Sans:wght@400;600;700&family=Poppins:ital,wght@0,300;0,400;0,500;0,600;0,700;1,600&family=Public+Sans:wght@300;400;700&family=Raleway:wght@400;700&family=Readex+Pro:wght@300;600&family=Roboto:wght@400;500&display=swap"
+         rel="stylesheet"
+      />
+
+      <link rel="stylesheet" href="style.css" />
+   </head>
+
+   <body>
+      <!-- Header  -->
+      <header class="header">
+         <nav class="header-nav">
+            <a href="#">About</a>
+            <a href="#">Work</a>
+            <a href="#">Contact</a>
```

Comment:
Wrap all links in a list format using the ul and li tags, as this is good practice

Reply:
- [ ] As I mentioned in my first comment. Please, always remember that your comments need to be polite as suggestions instead of order. For example: " Please , wrap all links ..."